src/target/stm32h7.c

```diff
@@ -273,6 +273,12 @@ bool stm32h7_probe(target_s *target)
 		}
 		break;
 	}
+	case ID_STM32H7Bx: {
```

Comment:
Unless you're intending to introduce new variables into the scope this creates, please drop the extra braces as they're not required. The code base only uses them for scope creation in case statements.

Reply:
There is a need for an extra `flash_size` scoped variable to support *both* H7A3/H7B3 and H7B0, so I'm keeping it until next commit (for a cleaner diff).